lib/pipeline/addon/new-pipeline/route.js

```diff
@@ -22,28 +22,41 @@ let precanStages = ()=> [{
 export default Route.extend({
   globalStore: service(),
   projectStore: service('store'),
-
+  cookies: service(),
   precanStages: precanStages(),
   model: function() {
     var globalStore = this.get('globalStore');
     var projectStore = this.get('projectStore');
     let projectModel = window.l('route:application').modelFor('authenticated.project');
+    let user = window.l('service:access').me;
```

Comment:
I get `undefined` from `this.get('access').me` if I use the normal service.
We may change the API to make it scoped to the user so that I don't have to access the user id.